content/guide/configuration/options-pages.md

```diff
@@ -0,0 +1,123 @@
+
+**Translatable Options:** This page is for all fields that should be translated for each language the website uses. When combined with the WordPress plugin [WPML](https://wpml.org/), these fields will output their value from the active display language. This is useful for generic content that the user may wish to set once for many components.
+
+Fields added to either of these pages from a feature, component, or post type will display in the WordPress back end under a new sub page that matches the name of the component, feature, or post type where they are registered.
+
+![Flynt Options Pages](images/options-pages.jpg)
+
+## Adding Fields to Options Pages
+
+It is possible to add fields to both of these pages using the `fields.json` file of a component or a feature. [Please take a moment to read through the content fields documentation](/guide/components/content-fields/) if you are unfamiliar with how this works.
+
+To show fields from a component or feature on either of the options pages, the fields must be defined in `fields.json` inside an array with the name of the options page.
+
+To show fields under the **global options** page, register fields inside the `globalOptions` array:
+
+```json
+{
+  "globalOptions": [
+    {
+      "name": "gaId",
+      "label": "Google Analytics ID",
+      "type": "text",
+      "maxlength": 20,
+      "prepend": "",
+      "append": "",
+      "placeholder": "XX-XXXXXXXX-X"
+    }
+  ]
+}
+```
+
+To show fields under the **translatable options** page, register the fields inside the `translatableOptions` array:
+
+```json
+{
+  "translatableOptions": [
+    {
+      "name": "brandLogo",
+      "label": "Brand Logo",
+      "type": "image",
+      "mime_types": "jpg,jpeg,svg"
+    }
+  ]
+}
+```
+
+If these fields are defined inside a component, this field data is already available in the [view template](/guide/components/view-templates/).
+
+If these fields are defined inside a feature or post type, or you need to use options of one component inside of another, it is necessary to use the `OptionPages` class provided by the [Acf](https://github.com/flyntwp/flynt-starter-theme/tree/master/Features/Acf) feature. With the `get` function of the `OptionPages` class, it is possible to get either a single option or all options of a sub page.
+
+For example, to get all options from the Google Analytics feature:
+
+```php
+<?php
+
+use Flynt\Features\Acf\OptionPages;
+
+$GoogleAnalyticsGlobalFields = OptionPages::get('globalOptions', 'feature', 'GoogleAnalytics');
+```
+
+To get only the `gaId` option from the Google Analytics feature:
+
+```php
+<?php
+
+use Flynt\Features\Acf\OptionPages;
+
+$GoogleAnalyticsId = OptionPages::get('globalOptions', 'feature', 'GoogleAnalytics', 'gaID');
```

Comment:
Also, not sure about this, but you mention the `gaId` option above and here it looks for the `gaID` option. Note the D. ;)